tests/unit/common/test_uploaded_file.py

```diff
@@ -21,36 +23,69 @@ def tearDown(self):
         super().tearDown()
         pass
 
-    def test_from_s3_key(self):
+    def _create_s3_object(self, filename,
+                          content_type="application/octet-stream",
+                          content="file_content"):
         s3 = boto3.resource('s3')
-        filename = "file1"
-        content_type = "application/octet-stream; dcp-type=data"
         s3_key = f"{self.upload_area_id}/{filename}"
         s3object = s3.Bucket(self.upload_config.bucket_name).Object(s3_key)
-        s3object.put(Body="file1_body", ContentType=content_type)
+        s3object.put(Body=content, ContentType=content_type)
+        return s3object
+
+    def test_from_s3_key__loads_data_and_creates_db_record_if_none_exists(self):
```

Comment:
nit was the double underscore (here and for `test_from_s3_key__loads_data_but_doesnt_creates_db_record_if_one_exists`) intentional?